local-cli/runAndroid/runAndroid.js

```diff
@@ -156,6 +156,20 @@ function buildAndRun(args) {
       'utf8'
     ).match(/package="(.+?)"/)[1];
 
+    const variantName = args.variant || 'debug';
```

Comment:
Use the current command `--configuration`?
Also, the `--variant` is not described in the `module.export.options`.